src/main/kotlin/net/irisshaders/lilybot/utils/_Utils.kt

```diff
@@ -42,7 +47,7 @@ suspend fun EphemeralSlashCommandContext<*>.getConfigPublicResponse(inputColumn:
 }
 
 /**
- * This is a simple function to get a value from the configuration database in an ephemeral message command context,
+ * This is a simple function to get a value from the configuration database in an [EphemeralMessageCommandContext],
  * null check it, and respond accordingly. It takes a string input of the column that is requested and gets it, if it's
  * null a private error message (i.e. for commands staff run), and null is returned. If null this should be handled with
```

Comment:
This should say public.

Reply:
It shouldn't. The name is perhaps confusing, but it this command is for ephemeral context, but provides a response that the public should recieve, rather than staff. This function will tell the user to tell a staff member, wheras private response will ask if the configuration is set